cps/src/main/js/workflow-editor.js

```diff
@@ -57,6 +57,19 @@ jenkinsJSModules.import('ace-editor:ace-editor-122')
                 }
                 showSamplesWidget();
             });
+            
+            // Make the editor resizable using jQuery UI resizable (http://api.jqueryui.com/resizable).
+            // ACE Editor doesn't have this as a config option.
+            $wfEditor.wrap('<div class="jquery-ui-1"></div>');
+            $wfEditor.resizable({
+                handles: "s", // Only allow vertical resize off the bottom/south border
+                resize: function() {
+                    editor.resize();
+                }
+            });
+            // Make the bottom border a bit thicker as a visual cue.
+            // May not be enough for some people's taste.
+            $wfEditor.css('border-bottom-width', '0.4em');
```

Comment:
We'd need to add css specific to this adjunct. We could do that, but I think I'd wait 'till we have more style.